src/stylesheets/components/_notFound.scss

```diff
@@ -7,7 +7,26 @@
     text-align: center;
     width: 100%;
 
+    .logo-image {
+        height: 150px;
+        width: 150px;
+        border-radius: 50%;
+    }
+
     .content {
-        margin: 15px;
+        margin: 30px 0 60px;
+    }
+
+    .button-container {
+        flex-direction: row;
+    }
+
+    .button {
```

Comment:
Can you make both buttons the same width? The different widths look awkward to me.

Reply:
Sure.